test/test.mk

```diff
@@ -30,7 +30,7 @@ test:
 	$(test-dir)/env -r $(base) $(mode) $(binaries) $(pkg-$(ARCH)) $(ninepm) -v $(TESTS)
 
 test-sh:
-	$(test-dir)/env $(base) $(mode) $(binaries) $(pkg-$(ARCH)) -i /bin/sh
+	$(test-dir)/env $(base) $(mode) $(binaries) $(pkg-$(ARCH)) -v $(INFIX_VERSION) -i /bin/sh
```

Comment:
We already supply the disk image and the package file, both of which have the version information contained in it. Could we not source it from there, rather than having to specify it again? I.e. have the meta test fish it out of the zip?

Reply:
The disk image supplied to the test system is a qcow2. To fish it out you need to convert it to raw  it with qemu-image, and then mount it on loopback. We could always supply the squash, even if we only use the qcow2 in the test?